qiita_pet/handlers/download.py

```diff
@@ -235,7 +235,7 @@ def get(self, study_id):
              (self.current_user in study.shared_with)))
 
         for a in study.artifacts(artifact_type='BIOM'):
-            if full_access or a.visibility == 'public':
+            if full_access or a.visibility == 'public' and not a.has_human:
```

Comment:
I think the logical grouping on this is the following. Why is there a limitation on `artifact_type='BIOM'` though?

```suggestion
            if full_access or (a.visibility == 'public' and not a.has_human):
```

Reply:
Good catch, missed that!